<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblOutput">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnCalculate">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="homeworkGrade">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>113</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="homeworkLabel">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Homework</string>
         </property>
        </widget>
        <widget class="QLabel" name="assignmentLabel">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Project Assignments</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="assignmentsGrade">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>113</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="exam01Label">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Exam 01</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="midtermGrade">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>113</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="finalGrade">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>90</y>
           <width>113</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="midtermLabel">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Midterm Project</string>
         </property>
        </widget>
        <widget class="QLabel" name="finalLabel">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>90</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Final Project</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="exam01Grade">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>113</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="exam02Label">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Exam 02</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="exam02Grade">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>50</y>
           <width>113</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCalculate"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalculate">
   <property name="text">
    <string>Calculate...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
